tools/lib/test_server.py

```diff
@@ -16,7 +17,11 @@
 except ImportError as e:
     print("ImportError: {}".format(e))
     print("You need to run the Zulip tests inside a Zulip dev environment.")
-    print("If you are using Vagrant, you can `vagrant ssh` to enter the Vagrant guest.")
+    username = getpass.getuser()
```

Comment:
I assume it was just overlooked when we made `sanity_check.py`, or maybe there was a PR pending.  Can you investigate how easy it is to consolidate them at this point?